source/support.tex

```diff
@@ -1249,8 +1249,8 @@
 The header also defines numerous macros of the form:
 
 \begin{codeblock}
-  INT_[FAST LEAST]{8 16 32 64}_MIN
-  [U]INT_[FAST LEAST]{8 16 32 64}_MAX
+  INT[_FAST _LEAST]{8 16 32 64}_MIN
+  [U]INT[_FAST _LEAST]{8 16 32 64}_MAX
   INT{MAX PTR}_MIN
```

Comment:
Thanks for the fix. Perhaps we could take this opportunity to further improve the presentation here, and shift away from this ambiguous and unexplained notation. Perhaps a grammar fragment:

> *min-max-macro:*
>> *signed-type* `_MIN`
>> *signed-type* `_MAX`
>> *unsigned-type* `_MAX`
>
> *signed-type:*
>> *sized-type-prefix* *size*
>> one of `INTMAX` `INTPTR` `PTRDIFF` `SIG_ATOMIC` `WCHAR` `WINT`
>
> *unsigned-type:*
>> `U` *sized-type-prefix* *size*
>> one of `UINTMAX` `UINTPTR` `SIZE_MAX`
>
> *sized-type-prefix:* one of `INT` `INT_FAST` `INT_LEAST`
>
> *size:* one of `8` `16` `32` `64`